tutorials/best_practices/scenes_versus_scripts.rst

```diff
@@ -50,8 +52,10 @@ a change in API:
         {
             _node = new Node();
             _myNode = MyNode.New().As<Node>();
-            _myScene = MyScene.Instantiate(); // Different. Instantiated from a PackedScene
-            _myInheritedScene = MyScene.Instantiate(PackedScene.GenEditState.Main); // Create scene inheriting from MyScene
+            // Different than calling new() or MyNode.New(). Instantiated from a PackedScene
```

Comment:
```suggestion
            // Different than calling new() or MyNode.New(). Instantiated from a PackedScene.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).

Reply:
Should I also fix the corresponding GDScript comments where applicable, or just stick to the updated C# comments?

Reply:
Can do them too where applicable :)

Reply:
I started looking over the files I'd modified for comment formatting, and spotted a chance to get rid of some foo/bar examples in the cross-language scripting tutorial. However, those go hand-in-hand with several spots where a comment is used to indicate the expected output of a print line (the raw string, no quotes). ([Here's one example](https://github.com/godotengine/godot-docs/blob/9ce9593ee38b008ca325ec277118318e49c37bbc/tutorials/scripting/cross_language_scripting.rst?plain=1#L121).) 

Two questions:
* Should those instances be updated, and if so, what would be the appropriate format for that? Something like `Expected output is "My Value 1".`?
* If they should be updated, should I open another PR for that? Seems like this issue is drifting away from the original purpose here.

Reply:
I'd say to deal with it separately indeed

Generally error prints are supposed to also end with a period, but it's far from uniform, and don't think it's relevant here, at least not to justify doing a sweeping fix